modules/cnn_3dobj/README.md

```diff
@@ -1,20 +1,20 @@
-#Convolutional Neural Networks for 3D object classification and pose estimation.
+Convolutional Neural Networks for 3D object classification and pose estimation.
 ===========================================================
-#Module Description on cnn_3dobj:
-####This module uses Convolutional Neural Networks (Caffe) to train and recognize 3D poses of objects using triplet networks. The main reference paper can be found at:
+# Module Description on cnn_3dobj:
+#### This module uses Convolutional Neural Networks (Caffe) to train and recognize 3D poses of objects using triplet networks. The main reference paper can be found at:
```

Comment:
This should be a regular text instead of heading-4

Reply:
Good idea